common/simple_junit/include/simple_junit/junit5.hpp

```diff
@@ -181,6 +181,8 @@ struct SimpleTestCase
 {
   Skipped skipped;
 
+  std::vector<Pass> pass;
```

Comment:
Please remove this unused and non-standard element `std::vector<Pass> pass`. 
Type `Pass` is not a valid JUnit 5 element.